install_config/configuring_azure.adoc

```diff
@@ -11,25 +11,43 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access a
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
+For more information about adding administrator roles, see link:https://docs.microsoft.com/en-us/azure/billing/billing-add-change-azure-subscription-administrator[Add or change Azure subscription administrators].
+
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Microsoft Azure Disk as a persistent volume on the {product-title} version 3.5 or later, you must enable Azure Cloud Provider.
+* All {product-title} node virtual machines (VMs) running in Microsoft Azure must belong to a single resource group.
+* Microsoft Azure VMs must be named the same as {product-title} nodes and this
```

Comment:
Yup, I think that's fine :+1: 

I believe this is because OpenShift uses the hostname of the VM to "lookup" the VM in Azure for further configuration. I'm not quite sure where/how this happens though.